.pipelines/templates/template-podman.yml

```diff
@@ -0,0 +1,174 @@
+parameters:
+  - name: jobName
+    type: string
+  - name: makeCommand
+    type: string
+  - name: repository
+    type: string
+  - name: acrFQDN
+    type: string
+  - name: imageRepoName
+    type: string
+  - name: pushLatest
+    type: boolean
+    default: false
+
+jobs:
+  - job: ${{ parameters.jobName }}
+    pool:
+      name: 1es-aro-ci-pool
+    variables:
+      ACR_FQDN: ${{ parameters.acrFQDN }}
+      REPOSITORY: ${{ parameters.repository }}
+      IMAGE_TAG: $(Build.BuildId)  # Include the unique BuildId or another identifier
+    steps:
+      - template: ../templates/template-checkout.yml
+
+      - task: AzureCLI@2
+        inputs:
+          azureSubscription: 'ado-pipeline-dev-image-push'  # Service connection name
+          scriptType: 'bash'
+          scriptLocation: 'inlineScript'
+          inlineScript: |
+            echo "Retrieving ACR token..."
+            REGISTRY_NAME=${ACR_FQDN%%.*}
+
+            if [ -z "$REGISTRY_NAME" ]; then
+              echo "Error: ACR_FQDN is not set correctly."
+              exit 1
+            fi
+            TOKEN=$(az acr login --name $REGISTRY_NAME --expose-token --output tsv --query accessToken)
+
+            if [ -z "$TOKEN" ]; then
+              echo "Error: Failed to retrieve ACR token."
+              exit 1
+            fi
+
+            echo "##vso[task.setvariable variable=ACR_TOKEN]$TOKEN"
+
+      - script: |
+          set -xe
+          export NO_CACHE=true
+          export TOKEN=$(ACR_TOKEN)
+          export REPOSITORY=${{ parameters.repository }}
+          export IMAGE_REPO_NAME=${{ parameters.imageRepoName }}
+
+          # Install podman, make, azure-cli, and Go
+          echo "Installing Podman, Make, Azure CLI, and Go..."
+          sudo rpm --import https://packages.microsoft.com/keys/microsoft.asc
+          sudo dnf config-manager --add-repo https://packages.microsoft.com/config/rhel/8/prod.repo
+          sudo dnf install -y azure-cli podman make
+
+          # Install Go
+          curl -OL https://golang.org/dl/go1.21.11.linux-amd64.tar.gz
+          sudo tar -C /usr/local -xzf go1.21.11.linux-amd64.tar.gz
+          export PATH=$PATH:/usr/local/go/bin
+
+          # Verify installations
+          which make
+          make --version
+          which podman
+          podman --version
+          which go
+          go version
+          whoami
+
+          # Source the podman control script
+          echo "Sourcing podman control script and starting podman service..."
+          . ./hack/e2e/utils.sh
+          run_podman
+          validate_podman_running
+
+          # Log in to ACR using Podman
+          echo "Logging into ACR with Podman..."
+          echo $TOKEN | podman --remote --url=tcp://localhost:8888 login ${ACR_FQDN} -u "00000000-0000-0000-0000-000000000000" --password-stdin --log-level=debug 2>&1 | tee podman_login.log
```

Comment:
I notice we're using `--remote --url=tcp://localhost:8888` throughout this - can we make that a standard parameter like we did in the Makefile? Alternatively, we may be able to configure podman's default configuration to point to that connection?